Cargo.toml

```diff
@@ -74,6 +75,11 @@ path = "sdk/core/azure_core_test_macros"
 # azure_identity should only ever be in dev-dependencies herein
 path = "sdk/identity/azure_identity"
 
+[workspace.dependencies.azure_storage_blob]
+# azure_storage_blob should only ever be in dev-dependencies herein
+path = "sdk/storage/azure_storage_blob"
+version = "0.4.0"
```

Comment:
Dependencies that are only dev-dependencies shouldn't have a version. But the fact you depend on this for your Storage Blobs checkpoint store means the comment is incorrect and misleading, right?